### PR TITLE
Still working on :ShadowTroll:

### PR DESCRIPTION
**Pull request by rodney528**

### CODE REVIEW

File: source/FunkinLua.hx [hunk 1]
```diff
@@ -920,15 +920,16 @@ class FunkinLua {
...
+			if (difficultyNum <= -1)
 				difficultyNum = PlayState.storyDifficulty;
-			// MEGA FIX DIFFS BETWEEN WEEKS WILL NOW FUCKING WORK WOOOOOOOOOOOOW
+			// MEGA FIX, DIFFS BETWEEN WEEKS WILL NOW FUCKING WORK WOOOOOOOOOOOOW
 			if (weekJson == null || weekJson.length < 1) {
-				weekJson = WeekData.weeksList[PlayState.storyWeek];
+				weekJson = WeekData.weeksList[PlayState.storyWeek].difficulties;
 			} else {
-				weekJson = PlayState.storyDifficulty;
+				weekJson = weekJson.difficulties;
```
Comment (kingyomoma) [inline source/FunkinLua.hx:929]: `String`'s don't have property `difficulties`

**Review by iym1337 — CHANGES_REQUESTED** [review]

File: source/FunkinLua.hx (hunk 1 above)
Comment (kingyomoma) [reply source/FunkinLua.hx:929]:
```haxe
var identifier;
if (weekJson == null || weekJson.length < 1) 
    identifier = WeekData.weeksList[PlayState.storyWeek];
else 
    identifier = weekJson;

if (identifier != null) 
    CoolUtil.difficulties = WeekData.weeksLoaded.get(identifier).difficulties;
```
(obviously untested)

Comment (ImaginationSuperHero528) [reply source/FunkinLua.hx:929]: Ok I'll test it tomorrow

Comment (ImaginationSuperHero528) [reply source/FunkinLua.hx:929]: I completely forgot about this 🤣

**Comment by ImaginationSuperHero528** [comment]
plz build 🤞🏻

**Comment by ImaginationSuperHero528** [comment]
fuck

**Pull request closed without merging**